packages/datadog-instrumentations/src/http/server.js

```diff
@@ -65,9 +72,7 @@ function wrapEmit (emit) {
           // TODO: should this always return true ?
           return this.listenerCount(eventName) > 0
         }
-        if (finishSetHeaderCh.hasSubscribers) {
```

Comment:
I added this here long time ago to prevent wrapping a method when noone is waiting for it. It is not the usual behaviour that we have in the tracer, so I decided to wrap the method always and check `finishSetHeaderCh` in the wrap, as we do always.